src/core/store/middleware.ts

```diff
@@ -184,19 +187,10 @@ export namespace Middleware {
     const composeEnhancers = global && global['__REDUX_DEVTOOLS_EXTENSION_COMPOSE__'] || compose;
 
     return composeEnhancers(
-      applyMiddleware(
-        batchMiddleware,
-        saveStateAnalyzer,
-        pastPurchaseProductAnalyzer
-      ),
-      applyMiddleware(
-        batchMiddleware,
-        ...middleware
-      ),
-      applyMiddleware(
-        batchMiddleware,
-        Middleware.validator
-      ),
+      applyMiddleware(...normalizingMiddleware, saveStateAnalyzer, pastPurchaseProductAnalyzer),
```

Comment:
Should attempt to reduce the amount of `normalizingMiddleware` we apply

Reply:
We should do this as part of the URL ticket since we'll be touching it anyway.